### PR TITLE
Add LeFishe Keyboard

### PR DESCRIPTION
**Pull request by lyso1**
## Description

65xt pcb for a 65xt keyboard. has 4 keymaps that have been tested. No rgb or led support is available on the pcb. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

re forked the repo qmk to have the most up to date branch for changes, and to make it so that my submodules weren't being updated our of no where.

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/lefishe/readme.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# lefishe
+
+firmware for the 65xt keyboard designed around the symmetrical blocker 65% layout.
+This pcb cab also be used as a replacement pcb for the revoKmini/kmacmini with some modification to the design
+
+* Keyboard Maintainer: [Lyso1](https://github.com/lyso1)
+* Hardware Supported: LeFishe, atmega32u4
+* Hardware Availability: [/u/TheLysol_27](https://www.reddit.com/user/TheLysol_27)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/lefishe/readme.md:10]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/lefishe/readme.md (hunk 1 above)
Comment (lyso1) [reply keyboards/lefishe/readme.md:10]: added instructions in the latest commit

File: keyboards/lefishe/lefishe.c [hunk 2]
```diff
@@ -0,0 +1 @@
+#include "lefishe.h"
```
Comment (drashna) [inline keyboards/lefishe/lefishe.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/lefishe/lefishe.c (hunk 2 above)
Comment (lyso1) [reply keyboards/lefishe/lefishe.c:17]: sure. added

**Review by drashna — APPROVED** [review]

File: keyboards/lefishe/readme.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+    make lefishe:default
+
+Enter into the bootloadder to flash new firmware in 3 ways:
```
Comment (fauxpark) [inline keyboards/lefishe/readme.md:14]:
```suggestion
Enter into the bootloader to flash new firmware in 3 ways:
```

File: keyboards/lefishe/readme.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+	* **Bootmagic reset**: Hold down the key at (0,0) in the matrix(The F1 key in this case) and plug the the keyboard in. 
+	* **Physical reset button**: Briefly press the button on the back and left side of the PCB(exactly under where the "2" key would be located) while the PCB is plugged in.
+	* **Keycode in layout**: Press the key mapped to `RESET'(RALT + E in this case) while the keyboard is plugged in.
```
Comment (fauxpark) [inline keyboards/lefishe/readme.md:18]:
```suggestion
	* **Keycode in layout**: Press the key mapped to `RESET` (RALT + E in this case) while the keyboard is plugged in. 
```

File: keyboards/lefishe/info.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
+{
+    "keyboard_name": "Le_Fishe", 
+    "url": "", 
+    "maintainer": "Lyso1", 
+    "width": 18.25, 
+    "height": 5,
```
Comment (fauxpark) [inline keyboards/lefishe/info.json:6]:
```suggestion
```

File: keyboards/lefishe/info.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+        }
```
Comment (fauxpark) [inline keyboards/lefishe/info.json:10]:
```suggestion
        },
```

File: keyboards/lefishe/info.json [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
```
Comment (fauxpark) [inline keyboards/lefishe/info.json:13]:
```suggestion
        },
```

File: keyboards/lefishe/info.json [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
```
Comment (fauxpark) [inline keyboards/lefishe/info.json:16]:
```suggestion
        },
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**